source/features/preview-hidden-comments.tsx

```diff
@@ -18,10 +17,11 @@ const init = (): void => {
 			continue;
 		}
 
-		const header = select(`
-			summary .timeline-comment-header-text,
-			summary .discussion-item-copy
-		`, details)!;
+		const header = select([
+			'summary .timeline-comment-header-text', // Issue and commit comments
+			'summary .discussion-item-copy',
+			'div.d-inline-block' // Review Comments
```

Comment:
This is so generic. Let's try
```suggestion
			'.discussion-item-icon  + div' // Review Comments
```